src/utils/url.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Check for a valid url string
+ *
+ * @param {string} textString — string to be checked
+ *
+ * @returns {boolean}
+ */
+export const url = function (textString) {
+  const regex = /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/;
```

Comment:
You can also do check this way:

```
try {
  new URL(textString);

  return true;
} catch (e) {
  // it's not an URL
}
```

Reply:
yes, but `test:hehehe` will be a url this way but it is not good i think. 

protocol must be in the url string